plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -207,7 +202,7 @@ func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes,
 		saInfo = serviceaccount.UserInfo(a.GetNamespace(), pod.Spec.ServiceAccountName, "")
 	}
 
-	matchedPolicies, err := c.pspMatcher(c.lister, a.GetUserInfo(), saInfo, c.authz, a.GetNamespace())
+	matchedPolicies, err := c.lister.List(labels.Everything())
```

Comment:
Consider renaming `matchedPolicies` to `allPolicies` or just `policies`.

Reply:
Done.